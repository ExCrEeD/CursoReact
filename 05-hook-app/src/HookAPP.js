import React from 'react'

export const HookAPP = () => {
    return (
        <div>
            <h1>Hola Mundo</h1>
        </div>
    )
}
