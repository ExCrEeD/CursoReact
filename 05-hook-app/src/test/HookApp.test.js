import { shallow } from "enzyme";
import { HookAPP } from "../HookAPP";

describe('prueba hook app', () => {
    test('debe renderizar el componente ', async () => {
        let wrapper = shallow(<HookAPP/>);
        expect(wrapper).toMatchSnapshot();
    })
    
})
