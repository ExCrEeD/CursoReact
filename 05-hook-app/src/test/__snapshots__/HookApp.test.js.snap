// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prueba hook app debe renderizar el componente  1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
